### ISSUE TITLE
Homeduino Problem - DS18B20 and RF won't work

### ISSUE THREAD

**Issue by beniroquai** [pimatic/pimatic#586]
I tested both, the temp-sensor and the transmitter seperatly with a standalone program on the pc. It worked quiet well. I got the ID of the Sensor and the Elro AB440 was controlled over remote as well. 
Now I followed the wiki on how to install the Homeduino with an Arduino Nano 328 (Actually Chinese, CH341S USB-Controller, does it matter?!)
I loaded 'git clone --recursive https://github.com/pimatic/homeduino.git', cd /homeduino and then changed Makefile accordingly. Befor 'make upload' I was also typed 'git checkout onewire'. But then an error occurs:

homeduino.ino:2:21: fatal error: OneWire.h: No such file or directory
compilation terminated.
/home/pi/homeduino/arduino-mk/Arduino.mk:1091: recipe for target 'build-nano328/homeduino.o' failed
make: *** [build-nano328/homeduino.o] Error 1

Previously I was able to compile and upload the code (without the Sensor), but the sockets won't work. Can I also upload the code via the Arduino envirornment? Perhaps the Pi doesn'T have the correct driver?

Thank you very much. Great project! :)

**Comment by sweetpi** [comment]
That's badly documented. I think you have to update the onewire git submodule, after checking out the onewire branch:

```
git submodule update --init --recursive
```

**Comment by beniroquai** [comment]
Awesome. This works! Next problem: Sensor Value stays at -127°C .. It's cold in here, but not that cold ;).. I tested it with a programm on the arduino itself and it worked. My Device setup looks like this:
   {
      "id": "homeduino-temperature-dst",
      "name": "DST",
      "class": "HomeduinoDSTSensor",
      "pin": 11,
      "address": "28FBBE8F500F",
      "interval": 1000
    },

Pin Number of the arduino is the physical adress, right? Baudrate is 115200? Sending and Recieving is still not possible. The Dip-Adress of the Sockets should be ok. I've tested it with a seperate programm..Maybe communication Problem between Pi and Arduino? Can I debug the communication somehow?

**Comment by beniroquai** [comment]
Hey. It was just a typo. The Adress was wrong. Sorry!


_________________________________________________________________________
Mit freundlichen Grüßen

benedictdied@gmail.com
Benedict Diederich

2015-03-13 7:55 GMT+01:00 sweet pi - sweet home <notifications@github.com>:

**Comment by tomansi** [comment]
Interval 1000 is too small. The sensors needs 750ms to read values. Comercial sensors sends values every 45000.

**Comment by leader21** [comment]
Indeed! Keep in mind that the update interval is in milliseconds.  So 1000ms is 1 second. better to use 60000 as interval = 1 minute.

**Comment by beniroquai** [comment]
Yep. Thanks! I updated the corresponding time value. But still facing the problem with the elro socket. Does it matter weather the reciever is connected or not? Any noise disturbing connection between transmitter/socket? I tried almost any protocoll and DIP Setting, but nothin happened. The remote is still broken ;)..I try to get it to work. 

Anyway great project! Especially the MAX! functionality! :+1:
